### PR TITLE
Updating Support Link in README

### PR DESCRIPTION
**Pull request by cpustejovsky**
What: 
community.storj.io/home tells me that this has moved to https://forum.storj.io// so I updated the link in the README to reflect that

Why:
It means new people coming to this repo (like me) have one less link to click when getting their bearings.

Please describe the tests:
 - Test 1: N/A (markdown update)
 - Test 2: N/A (markdown update)
 
Please describe the performance impact: No performance impact; markdown update

## Code Review Checklist (to be filled out by reviewer)
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,5 +118,5 @@ code under Apache v2, or other licenses in the future.
 # Support
 
 If you have any questions or suggestions please reach out to us on
-[Rocketchat](https://community.storj.io/) or
+[Rocketchat](https://https://forum.storj.io/) or
```
Comment (ifraixedes) [inline README.md]:
```suggestion
[our community forum](https://https://forum.storj.io/) or
```

The forum isn't a "Rocketchat" anymore.

**Comment by cpustejovsky** [comment]
@ifraixedes I signed the CLA agreement. Apologies for not seeing that before I made the PR

**Comment by ifraixedes** [comment]
@cpustejovsky I'm not sure if you've realized that I suggested a change, can you apply it or reply, please?

File: README.md (hunk 1 above)
Comment (cpustejovsky) [reply README.md]: Fixed, thanks! :+1:

**Comment by cpustejovsky** [comment]
@ifraixedes fixed typo changes, thanks for the feedback! :+1:

**Pull request merged**